roles/sap_vm_provision/tasks/platform_ansible/gcp_ce_vm/execute_provision.yml

```diff
@@ -128,12 +129,6 @@
     auth_kind: "serviceaccount"
     service_account_file: "{{ sap_vm_provision_gcp_credentials_json }}"
 
-# Required as state: present on Ansible Module gcp_compute_instance does not allow for waiting until VM has booted
```

Comment:
Would retain this comment and place above the `wait_for_connection` replacement code?